src/lightning_lite/utilities/rank_zero.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import os
+import warnings
+from functools import partial, wraps
+from platform import python_version
+from typing import Any, Callable, Optional, Union
+
+import lightning_lite as lite
+
+log = logging.getLogger(__name__)
+
+
+def _get_rank(strategy: Optional["lite.strategies.Strategy"] = None) -> Optional[int]:  # type: ignore[name-defined]
+    if strategy is not None:
+        return strategy.global_rank
```

Comment:
This is the only implementation change. The utility in PL relies on trainer here.